src/lib/salad/hll_empirical.c

```diff
@@ -0,0 +1,5362 @@
+#include "hll_empirical.h"
+
+#include <trivia/util.h>
+
+#define DATA_POINTS  200
+
+/*
+ * Estimation data that is used to determine the bias correction.
+ * For example, if raw estimation value lies on segment from
+ * estimation_data[p][idx] to estimation_data[p][idx+1] the
+ * corresponding bias correction lies on segment from
+ * estimation_bias_data[p][idx] to estimation_bias_data [p][idx+1]
+ * and can be interpolated. For more details see http://goo.gl/iU8Ig.
```

Comment:
I can't open your link.

Reply:
Started to use interpolated curves and removed this code.